backend/api/Services/MissionSchedulingService.cs

```diff
@@ -48,13 +47,18 @@ public async Task StartMissionRunIfSystemIsAvailable(string missionRunId)
             }
 
             try { await StartMissionRun(missionRun); }
-            catch (MissionException ex)
+            catch (Exception ex) when (
+                ex is MissionException
+                    or RobotNotFoundException
+                    or RobotNotAvailableException
+                    or MissionRunNotFoundException
+                    or IsarCommunicationException)
             {
                 const MissionStatus NewStatus = MissionStatus.Failed;
-                logger.LogWarning(
-                    "Mission run {MissionRunId} was not started successfully. Status updated to '{Status}'.\nReason: {FailReason}",
+                logger.LogError(
+                    ex,
+                    "Mission run {MissionRunId} was not started successfully due to {ErrorMessage}",
                     missionRun.Id,
-                    NewStatus,
```

Comment:
Why was the NewStatus part of the error removed? It could be nice to have in the log when debugging

Reply:
Mainly because it is not a dynamic variable at this point and always fixed as "Failed". Thus it seemed a bit redundant to me because we're already saying that the mission was not started successfully.